packages/docs/.vuepress/components/Toast.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <aside
+    v-if="active"
+    :class="[
+      'ods-toast',
+      'is-ods-toast-' + variant,
+      dismissed && 'is-toast-dismissed'
+    ]"
+  >
+    <span class="ods-toast--icon">
+      <svg
+        viewBox="0 0 14 14"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        class="ods-icon"
+        aria-hidden="true"
+      >
+        <path
+          v-if="variant"
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          :d="icon[variant]"
+          fill="currentColor"
+        />
+      </svg>
+    </span>
+    <h1 class="ods-toast--title">{{ title }}</h1>
+    <p class="ods-toast--body">{{ body }}</p>
```

Comment:
These toasts won't be read aloud to screen readers without a bit more scripting to handle the `aria-live` region! Something we can followup on

Reply:
Oh! The markup has this covered, but the Vue component doesn't. I'll add `role="status"` now; that ensure these get picked up by SRs when added to the DOM.

Reply:
Added `role="status"` to the demo component. There's also some guidance on this in the documentation.

Currently, the entire Toast will be treated as a live status, meaning the whole thing will be read. This may not be ideal when Toasts have too much content, but it feels like a better solve than forcing SR users to interact with something transient to get the rest of the content.

@arnoldsandoval-okta I know you may have some thoughts here as well.